src/errors.ts

```diff
@@ -386,3 +386,15 @@ export class NanoContractTransactionParseError extends Error {
 export class OracleParseError extends Error {
   errorCode: string = ErrorMessages.NANO_ORACLE_PARSE_ERROR;
 }
+
+export class GllTaskError extends Error {
```

Comment:
I'd rename it to `GlobalLoadLockerTaskError`.

Reply:
Done

Reply:
Thanks!